src/NuGet.Core/NuGet.Packaging/StringFormatter.cs

```diff
@@ -20,5 +20,17 @@ internal static class StringFormatter
                 source,
                 contentHash);
         }
+
+        internal static string ZipFileTimeStampModified(
+            string entry,
```

Comment:
👍  as long as it's not "entry", because someone looking at this method without the context of knowing where it's being used may not undersatnd what "entry" means.